en/deployment/articles/02-configuring-liferay/04-liferay-portal-clustering/05-enabling-cluster-link.markdown

```diff
@@ -130,87 +130,91 @@ from using UDP Multicast clustering, you can configure TCP Unicast. You must use
 this if you have a firewall separating any of your nodes or if your nodes are in
 different geographical locations.
 
-1.  Add a parameter to your app server's JVM:
+1.  Add a parameter to your app server's JVM on each node:
 
-        -Djgroups.bind_addr=[node_address]
- 
-    Use the node's IP address. 
+    ```bash
+    -Djgroups.bind_addr=[node_ip_address]
+    ```
 
-2.  Now you must determine the discovery protocol the nodes should use to
-    find each other. You have four choices: 
+2.  Select a discovery protocol for the nodes to use to find each other. Here are the protocol choices: 
 
     - TCPPing
     - JDBCPing
     - S3_Ping
     - Rackspace_Ping
 
-    If you aren't sure which one to choose, use `TCPPing`. This is used in the
-    rest of these steps; the others are covered below. 
+    If you aren't sure which one to choose, use TCPPing. The rest of these steps use TCPPing
+
+3.  Extract the `tcp.xml` file from `$LIFERAY.HOME/osgi/marketplace/Liferay Foundation - Liferay Portal - Impl.lpkg/com​.​liferay​.​portal​.​cluster​.​multiple​-​[version].​jar/lib​/​jgroups​-​[version].​Final​.​jar/tcp.xml` to a location accessible to DXP, such as a folder called `ehcache` in the DXP web application's `WEB-INF/classes` folder.  
```

Comment:
@dantewang I see your point about the folder name `ehcache`. Since the tcp.xml is all about TCP and has nothing to do with Ehcache, I'll rename it. I think I'll rename the folder to `jgroups`.

Reply:
Renamed the folder in https://github.com/tinatian/liferay-docs/pull/8/commits/12e631090ffd784dd1b8f426a04d7b72fe63c750